app/serializers/api/product_serializer.rb

```diff
@@ -49,10 +49,17 @@ class Api::CachedProductSerializer < ActiveModel::Serializer
   has_many :images, serializer: Api::ImageSerializer
   has_one :supplier, serializer: Api::IdSerializer
 
+  #return an unformatted descripton
   def description
     strip_tags object.description
   end
 
+  #return a sanitized html description
+  def description_html
+    d = sanitize(object.description, options = {tags: "p, b, strong, em, i"})
```

Comment:
Sorry sorry sorry, should have picked this up earlier. 

IMHO we should be sanitising input before it hits the database. What do you think @Matt-Yorkley?